src/services/api.tsx

```diff
@@ -130,23 +131,20 @@ export const getPatientsPerQuery = async (
    * TODO : for now, this function is limited by the server response to 500 resources. Must find a way to improve it (searching computation done by the server ideally ?)
    */
   let bundles: Bundle[] = [];
-  let params: string;
+  let params: string = "";
```

Comment:
nit: I think you don't need to precise the type if you assign ""

Reply:
Type removed